### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**
- [DEMO LINK](https://yelnikov-andrii.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://yelnikov-andrii.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  box-sizing: inherit;
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__img img {
```
Comment (SavMary) [inline src/style.css:34]: You should use class instead `img` (checklist 1)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  width: 160px;
+  height: 134px;
+}
+
+.card__description.bold {
```
Comment (SavMary) [inline src/style.css:39]: You can use only one class without `bold`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  outline: none;
+  color: #fff;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+}
+
+.link:hover {
```
Comment (SavMary) [inline src/style.css:131]: hover effect chenges card height

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,94 @@
...
+          <a 
+          href="#" 
+          class="link"
+          data-qa="hover">
+            Купить
+          </a>
+        </div>
+      </div>    
+
+      <div class="card">
```
Comment (SavMary) [inline src/index.html:53]: You can remove this card

Comment (SavMary) [inline src/index.html:53]: You can remove this card

File: src/style.css (hunk 2 above)
Comment (SavMary) [inline src/style.css:39]: You can use only one class without bold

File: src/style.css (hunk 3 above)
Comment (SavMary) [inline src/style.css:131]: hover effect chenges card height

File: src/style.css (hunk 1 above)
Comment (SavMary) [inline src/style.css:34]: You should use class instead img (checklist 1)

**Review by SavMary — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,52 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card"
```
Comment (Anna-Tkachenko) [inline src/index.html:12]: Don't create additional containers, card should be BEM block

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,52 @@
...
+           data-qa="card">
+        <img class="img container__card--img" src="./images/imac.jpeg" alt="image">
```
Comment (Anna-Tkachenko) [inline src/index.html:14]: Don't create additional BEM blocks (img): only card and stars

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,52 @@
...
+        Код товара: 195434
+        </span>
+        <div class="evaluation">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:28]: 🔥

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,52 @@
...
+          <div class="feedbacks">
+            Отзывов: 5
+          </div>
+        </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:32]: Use semantic tags

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,52 @@
...
+          <div class="value">
+            69 999 грн
+          </div>
+        </div>
+        <div class="btn">
+          <a 
+          href="#" 
+          class="link"
+          data-qa="hover">
+            Купить
```
[the remark this answers is not in the document]
Comment (Anna-Tkachenko) [reply src/index.html:46]: make it the element of card block

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  background: #e5e5e5;
+}
+
+.card {
+  width: 200px;
```
Comment (Anna-Tkachenko) [inline src/style.css:13]: You can't set width to the BEM block
Create additional class to make this block the element of body

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+  flex-direction: column;
+  background-color: #fff;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: flex;
+  margin-bottom: 40px;
+  width: 160px;
```
Comment (Anna-Tkachenko) [inline src/style.css:24]: Set left and right padding = 3px for image and width = 100% to pass the tests

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  display: flex;
+  justify-content: center;
```
Comment (Anna-Tkachenko) [inline src/style.css:40]: You don't have any child elements in card__title, delete redundant styles

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```
Comment (Anna-Tkachenko) [inline src/style.css:72]: Don't forget to remove margin for the last star

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
You didn't fix some changes
1) Add border to the card (check correct styles in figma)
2) You can't set width to the card BEM block, you should create page__card element for setting this styles
3) Add padding: 3px and width: 100% for image to pass tests
4) Remove border from the title
5) Check correct font-weight for 'Цена' text
6) You didn't add 400 and 700 font-weight for Roboto font

File: src/index.html [hunk 14]
```diff
@@ -1,13 +1,50 @@
...
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img" src="./images/imac.jpeg" alt="image of device">
+      <h3 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+      <span class="card__code">
+          Код товара: 195434
+      </span>
```
Comment (Anna-Tkachenko) [inline src/index.html:18]: Fix indents
```suggestion
      <span class="card__code">
        Код товара: 195434
      </span>
```

File: src/index.html [hunk 15]
```diff
@@ -1,13 +1,50 @@
...
+      </p>
+      <div class="card__btn">
+        <a 
+          href="#" 
+          class="card__btn-link"
+          data-qa="hover"
+        >
+          Купить
+        </a>
+      </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:47]: Don't create additional wrappers

File: src/style.css (hunk 10 above)
Comment (Anna-Tkachenko) [reply src/style.css:13]: You can't set width to the BEM block

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! 👍

**Pull request closed without merging**